IdentityCore/src/webview/embeddedWebview/ui/MSIDWebviewUIController.h

```diff
@@ -40,6 +41,9 @@ NSWindowController
 #if TARGET_OS_IPHONE
 @property (weak) UIViewController *parentController;
 @property UIModalPresentationStyle presentationType;
+
++ (WKWebViewConfiguration *)defaultWKWebviewConfiguration;
```

Comment:
Any reason not declare it as a property?

Reply:
That way we don't have to initialize the webview controller when calling it in the ADAL public API :)

Reply:
Maybe Kai meant a class property :)?